pyproject.toml

```
@@ -91,7 +94,7 @@ repair-wheel-command = [
 ]
 
 [tool.cibuildwheel.windows]
-before-build = "pip install delvewheel"
+before-build = "pip install delvewheel numpy"
```

I don't think this should be needed as we already provide it in the `requires` of the `[build-system]`.

```suggestion
before-build = "pip install delvewheel"
```